src/main/java/org/broadinstitute/consent/http/service/DarCollectionService.java

```diff
@@ -75,29 +84,227 @@ public DarCollectionService(DarCollectionDAO darCollectionDAO, DarCollectionServ
    * @param userRole The role the user is making the request as
    * @return List of DarCollectionSummary objects
    */
-  public List<Object> getSummariesForRoleName(User user, String userRole) {
-    switch (userRole) {
-      case Resource.ADMIN:
-        // TODO: Find all
+  private void updateStatusCount(Map<String, Integer> statusCount, String status) {
```

Comment:
nit: the javadoc that precedes this is not associated to the method.

Reply:
Minor style nit. I think most of these private void methods could be in a `DarCollectionSummary` utility class instead of sprinkled here in the service class.